RocketChat/Rocket.Chat.ReactNative
# feat: add `readThreads` params to `subscriptions.read`

## Proposed changes


Add `readThreads` params to `subscriptions.read` API for when doing the quick action of reading threads the app reads all messages and messages inside threads.

## Issue(s)	


## How to test or reproduce


In any channel

- create a thread

- send message inside of it with user A.

- Send it inside and outside the thread.

- Verify that user B has unread messages on that channel

- With user B, mark that channel as read.

- Verify that all messages are correctly marked.

## Screenshots

## Types of changes




## Checklist



## Further comments


Jira: [TC-912]


[TC-912]: https://rocketchat.atlassian.net/browse/TC-912?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
